src/types/annotator.js

```diff
@@ -17,8 +17,8 @@
  * @typedef DocumentMetadata
  * @prop {string} title
  * @prop {Object[]} link
- *   @prop {string} link.rel
- *   @prop {string} link.type
+ *   @prop {string} [link.rel]
+ *   @prop {string} [link.type]
  *   @prop {string} link.href
```

Comment:
Although all `<link>` tags should have a `rel` attribute, it appears that the `link` array contains entries from sources other than `<link>`  and those may only have an `href` property.